{
  "comments": [
    {
      "key": {
        "uuid": "77ba71a0_9d2d9f0b",
        "filename": "src/kits/media/BufferCache.cpp",
        "patchSetId": 1
      },
      "lineNbr": 44,
      "author": {
        "id": 1000011
      },
      "writtenOn": "2019-08-14T22:13:38Z",
      "side": 1,
      "message": "needless {}",
      "revId": "2362e2000f32156278db67c649404c0852a7547f",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "ec0b38db_f990516b",
        "filename": "src/kits/media/BufferCache.cpp",
        "patchSetId": 1
      },
      "lineNbr": 51,
      "author": {
        "id": 1000011
      },
      "writtenOn": "2019-08-14T22:13:38Z",
      "side": 1,
      "message": "needless newline",
      "revId": "2362e2000f32156278db67c649404c0852a7547f",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "7ecfec9e_2a64537e",
        "filename": "src/kits/media/BufferCache.cpp",
        "patchSetId": 1
      },
      "lineNbr": 61,
      "author": {
        "id": 1000002
      },
      "writtenOn": "2019-08-14T23:10:53Z",
      "side": 1,
      "message": "Allocating with `std::nothrow` will return NULL on allocation failure",
      "revId": "2362e2000f32156278db67c649404c0852a7547f",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "a7fd027d_f363aad6",
        "filename": "src/kits/media/BufferCache.cpp",
        "patchSetId": 1
      },
      "lineNbr": 65,
      "author": {
        "id": 1000002
      },
      "writtenOn": "2019-08-14T23:10:53Z",
      "side": 1,
      "message": "Given you\u0027re using exceptions, the above should not use `std::nothrow`, or this should be changed to a NULL check instead.",
      "revId": "2362e2000f32156278db67c649404c0852a7547f",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5",
      "unresolved": true
    }
  ]
}